nebullvm/inference_learners/openvino.py

```diff
@@ -60,15 +62,19 @@ class OpenVinoInferenceLearner(BaseInferenceLearner, ABC):
 
     def __init__(
         self,
-        exec_network,
+        model: openvino.pyopenvino.Model,
+        compiled_model: openvino.runtime.ie_api.CompiledModel,
+        infer_request: openvino.runtime.ie_api.InferRequest,
```

Comment:
This is an internal way of importing objects without additional enhancements. I would suggest to write it like this:
```suggestion
        model: openvino.runtime.Model,
        compiled_model: openvino.runtime.CompiledModel,
        infer_request: openvino.runtime.InferRequest,
```
This is an actual public API with documentation provided online.

Reply:
Thanks @jiwaszki for the suggestion. I'd actually ended up using the type inference based on the output of `type` on the respective objects.